original/shortkey.en.json

```diff
@@ -59,6 +59,7 @@
   "group_default_description": "see the books members of the group %{groupName} want to give, lend or sell",
   "group_invite_notification": "you have been invited to join a group",
   "group_invitor": "%{username} invited you to join",
+  "group_join_request_notification": "A group in which you are an admin has received a join request",
```

Comment:
suggestions to replace `group_join_request_notification` and the following `group_join_request_button`
- As admin of the group ${groupname}, you may accept or refuse %{requesterUsername}'s request.
- Go to group settings

Reply:
what about
- `group_join_request_notification`: `%{requesterUsername} requested to join the group ${groupname}`
- `group_join_request_button`: `Go to group settings to answer %{requesterUsername}'s request`

?

Reply:
`group_join_request_subject` & `group_join_request_notification` would state the same thing then? and we loose the why one's receive the email (admin). 
Its only the formulation `a group in which you are an admin has received`, as it's the group who receive the request (to be accepted or not) but the admin.

what about:
    `group_join_request_subject`: `%{requesterUsername} requests to join the group %{groupName}`,
    `group_join_request_notification`: `As admin of the group ${groupname}, you may decide to accept or refuse %{requesterUsername}'s request`
    `group_join_request_button`: `Go to group settings to answer %{requesterUsername}'s request`
?

Reply:
Wait, forget my suggestion, `group_join_request_notification` is the text to be displayed in the notifications settings (https://inventaire.io/settings/notifications), there is no `groupName` or `requesterUsername` available, it's just describing the general situation in which an email should be received.

But in the [current implementation](https://github.com/inventaire/inventaire/blob/b2938851888a433add04adba3acab215f6c09377/server/lib/emails/email.js#L122-L125), the email subject and the title in the email body are indeed the same: I guess a new key could be added to make them different?

Reply:
done in 45e51a0